### PR TITLE
Single click activation on projects popover

### PR DESCRIPTION
**Pull request by christopher-l**
These are not really clear improvements, but changes I like. The changes are:
- Entries are activated by single click (except when clicking the favorite checkboxes).
- Entries are selected (highlighted) on hover for visual cue.
- Entries are separated with grid lines.
- Paddings and size slightly changed.
- Font size for filename reduced.

What I would have liked, but not yet done is to include a style sheet to
- handle font sizes,
- make directory names slightly grayed out as in Gedit, and
- handle hover effects, maybe theme dependent to be more subtle than the selection effect.

If you like some of the changes, I am happy to adapt the pull request.

### CODE REVIEW

**Comment by mcepl** [comment]
Just a question: isn't the single-click / double-click activation somehow configurable? Or it isn't hard-coded in this script, is it?

File: src/project.rs [hunk 1]
```diff
@@ -73,6 +73,9 @@ impl Projects {
 
         projects.setup_tree();
 
+        projects.tree.set_activate_on_single_click(true);
```
Comment (christopher-l) [inline src/project.rs]: >  isn't the single-click / double-click activation somehow configurable? Or it isn't hard-coded in this script, is it?

At the moment, this is just hard-coded here (or implicitly hard-coded to require double clicks when the line is omitted). Personally, I would just always expect this kind of menu to work with single clicks, after all there is nothing else a single click would do here. So a configuration option would probably not make too much sense.

**Comment by mcepl** [comment]
OK, with applying this PR, single-click activation works for me. Thank you.

**Comment by mcepl** [comment]
So, what's remain this to be merged? I have rebased it on the top of the current ``master`` (ec22081) and it works just fine.

**Comment by christopher-l** [comment]
Sorry for the late answer, overlooked your question the first time.

 > Is there some kind of the key shortcut to get this dialogue opened?

Currently there isn't as far as I know, but I would like to have shortcuts for GUI functions, too. I think I'll look into this when I find some time.

> So, what's remain this to be merged?

I looked again for ways to change the styling of the `TreeView` elements and didn't really find a good solution for what I initially wanted. So I would leave it for now as is.

@daa84: Would you have something changed before merging?

**Comment by daa84** [comment]
Looks ok, thanks :+1:

**Pull request merged**